lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -14,13 +14,25 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
   @impl true
   def update(assigns, socket) do
     socket = assign(socket, assigns)
-
     secret_name = socket.assigns[:prefill_secret_name]
+    secret_value = socket.assigns[:prefill_secret_value]
+    session = socket.assigns[:session]
+    options = [{assigns.hub.id, "in #{hub_label(assigns.hub)}"}]
+
+    hub_id = if session, do: "session", else: assigns.hub.id
+    options = if session, do: [{"session", "in Session"} | options], else: options
```

Comment:
Since we are on the hub page, it doesn't really make sense to show the radio altogether.

I'm leaning towards just having a separate component instead of reusing this one, especially that we just need the form bit. This way we don't need to add more conditional paths, wdyt?

Reply:
I tried to re-use the existing component, but yeah, we should use one specific for the Hub (since we'll replicate for other hubs in the future)